modules/nw-ovn-kubernetes-list-resources.adoc

```diff
@@ -25,58 +25,59 @@ $ oc get all,ep,cm -n openshift-ovn-kubernetes
 .Example output
 [source,terminal]
 ----
-NAME                       READY   STATUS    RESTARTS      AGE
-pod/ovnkube-master-9g7zt   6/6     Running   1 (48m ago)   57m
-pod/ovnkube-master-lqs4v   6/6     Running   0             57m
-pod/ovnkube-master-vxhtq   6/6     Running   0             57m
-pod/ovnkube-node-9k9kc     5/5     Running   0             57m
-pod/ovnkube-node-jg52r     5/5     Running   0             51m
-pod/ovnkube-node-k8wf7     5/5     Running   0             57m
-pod/ovnkube-node-tlwk6     5/5     Running   0             47m
-pod/ovnkube-node-xsvnk     5/5     Running   0             57m
+NAME                                         READY   STATUS    RESTARTS      AGE
+pod/ovnkube-control-plane-55d6cb89fb-69cjj   2/2     Running   0             47m
+pod/ovnkube-control-plane-55d6cb89fb-7dmcd   2/2     Running   2 (37m ago)   47m
+pod/ovnkube-control-plane-55d6cb89fb-zkbqs   2/2     Running   0             47m
+pod/ovnkube-node-24tn8                       10/10   Running   0             34m
```

Comment:
ovnkube-node pods has 8 containers, here show 10. So this is not the latest version.

Reply:
Not sure what to do with the 9. Should I add a `NOTE` admonition to point out that Azure will have 9?

Reply:
I added a note in the docs.